Gems/AWSClientAuth/Code/Source/Authorization/ClientAuthAWSCredentials.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Authorization/ClientAuthAWSCredentials.h>
+
+#include <AzCore/RTTI/BehaviorContext.h>
+#include <AzCore/RTTI/AzStdOnDemandReflection.inl>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/AZStdContainers.inl>
+
+namespace AWSClientAuth
+{
+void ClientAuthAWSCredentials::Reflect(AZ::ReflectContext *context)
+{
+    auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
```

Comment:
The function `ClientAuthAWSCredentials::Reflect` is nested inside a namespace, it's missing tabulation.

Reply:
Reformatted this file in 4911a56

Reply:
Thanks @nemerle 
Approved!